File: app/src/main/java/com/infy/dagger2implementation/base/BaseFragment.java
package com.infy.dagger2implementation.base;

import android.content.Context;
import android.support.v4.app.Fragment;

import dagger.android.support.AndroidSupportInjection;

public class BaseFragment extends Fragment {

    @Override
    public void onAttach(Context context) {
        super.onAttach(context);
        AndroidSupportInjection.inject(this);

    }
}

File: app/src/main/java/com/infy/dagger2kotlin/base/BaseFragment.kt
package com.infy.dagger2kotlin.base

import android.content.Context
import android.support.v4.app.Fragment

import dagger.android.support.AndroidSupportInjection

class BaseFragment : Fragment() {

    override fun onAttach(context: Context?) {
        super.onAttach(context)
        AndroidSupportInjection.inject(this)

    }
}
